src/components/Profile/components/VictoryPromise/components/GraphDisplay.tsx

```diff
@@ -21,9 +21,10 @@ const GraphDisplay = ({ scores }: Props) => {
   Object.entries(scores).forEach((score) => {
     const [key, value] = score as [VictoryPromiseCategory, number];
     const index = GraphOrder[key];
+    const colorHex = (Colors[key].match(/#[A-Fa-f0-9]{6}/) || [''])[0];
```

Comment:
FYI a color hex code can have 8 digits if you include transparency.
```suggestion
    const colorHex = (Colors[key].match(/#[A-Fa-f0-9]{6,8}/) || [''])[0];
```

Reply:
@jsenning @Luke-W-Hart -- Saw this get merged at the same time as I commented